server/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -189,6 +191,10 @@
     private static final long NINETY_PER_HEAP_SIZE = (long) (JvmInfo.jvmInfo().getMem().getHeapMax().getBytes() * 0.9);
     private static final BytesReference EMPTY_BYTES_REFERENCE = new BytesArray(new byte[0]);
 
+    public static final String FEATURE_PREFIX = "client.features";
```

Comment:
why did you chose the "client" prefix? maybe use "transport.features" instead?

Reply:
I pushed 1f6682970001ac2430ece164630e74493ffc820b.